rock-paper-scissors/scripts/rock-paper-scissors.js

```diff
@@ -0,0 +1,158 @@
+const gameChoices = ["rock", "paper", "scissors"];
```

Comment:
FYI: Constants tend to be in singular capitalised snake case e.g. `GAME_CHOICE`

Singular because then when you select one of them (e.g. `GAME_CHOICE[1]`), you read it as "Game Choice 1"